test/Datadog.Trace.Tests/StopwatchTests.cs

```diff
@@ -0,0 +1,30 @@
+using System.Diagnostics;
```

Comment:
Can you move this file into a `Util` folder and namespace? Just to keep it consistent with `StopwatchHelpers` in `Datadog.Trace`.

Reply:
Done